app/src/main/java/io/appium/uiautomator2/server/ServerConfig.java

```diff
@@ -16,10 +16,106 @@
 
 package io.appium.uiautomator2.server;
 
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
 public class ServerConfig {
-    private final static int PORT = 6790;
+    // server port (default: `6790`)
+    private final static int SERVER_PORT =
+        System.getenv("SERVER_PORT") == null ? 6790 :
```

Comment:
these all ternary operators could be moved to a helper method

Reply:
This will be resolved w/ the upcoming update(s)